arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -233,40 +290,32 @@
             "$ref": "#/parameters/apiVersionParameter"
           },
           {
-            "name": "resourceGroupName",
-            "description": "Name of the resource group within the Azure subscription.",
-            "in": "path",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
-            "name": "mediaServiceName",
-            "description": "Name of the Media Service.",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "pattern": "^[a-z0-9]",
-            "minLength": 3,
-            "maxLength": 24
+            "$ref": "#/parameters/mediaResourceNameParameter"
           }
-        ]
-      },
-      "patch": {
-        "summary": "Update a Media Service.",
-        "operationId": "MediaService_Update",
+        ],
         "responses": {
           "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/MediaService"
-            }
+            "description": "Success. The Media Services account was deleted."

```

Comment:
As per [RFC2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html)

9.7 DELETE

The DELETE method requests that the origin server delete the resource identified by the Request-URI. This method MAY be overridden by human intervention (or other means) on the origin server. The client cannot be guaranteed that the operation has been carried out, even if the status code returned from the origin server indicates that the action has been completed successfully. However, the server SHOULD NOT indicate success unless, at the time the response is given, it intends to delete the resource or move it to an inaccessible location.

**A successful response SHOULD be 200 (OK) if the response includes an entity describing the status**, 202 (Accepted) if the action has not yet been enacted, or 204 (No Content) if the action has been enacted but the response does not include an entity.


Which means the 200 response should have a schema. Otherwise what is the difference between a 204 and a 200 ?